src/main.rs

```diff
@@ -335,6 +335,7 @@ fn main() -> anyhow::Result<()> {
       jump_back,
       base_url,
     } => {
+      println!("Partial quicksync is considered to be beta feature for now");
```

Comment:
Maybe make it a warning:
```suggestion
      println!("Warning: partial quicksync is considered to be beta feature for now");
```

And perhaps ask for confirmation like:
```rust
    println!("Do you want to continue? [y/N] ");

    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    if !input.trim().eq_ignore_ascii_case("y") {
        return Ok(());
    }
```

Reply:
Given that one read the manual and figured out that the CLI exists, I think it's safe to assume that they don't run it "randomly" to add the warning.